lib/src/list.dart

```diff
@@ -144,6 +145,10 @@ class ManagedRealmList<T extends Object?> with RealmEntity, ListMixin<T> impleme
     return result;
   }
 
+  void move(int oldIndex, int newIndex) {
```

Comment:
This method `move` is not visible in the API doc neither for List or RealmList. It is visible on the class `ListExtension` and it is not documented. Is it possible to have in the documentation `move` method described for `RealmList` and `List`?